.drone.yml

```diff
@@ -0,0 +1,95 @@
+kind: secret
+name: artifactory_access_token
+get:
+  path: artifactory-token?scope=artifactory-dv
+---
+kind: pipeline
+type: docker
+name: publish-packages
+depends_on:
+  - backstage-test
```

Comment:
This pipeline doesn't exist in your repo so I think you can probably just remove the `depends_on` block